ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -61,8 +59,9 @@ SQUtils.QObject {
 
     signal finished(int flow)
 
-    function init() {
-        root.stackView.push(entryPage)
+    function restart() {
+        clear()
+        replace(null, initialComponent)
```

Comment:
```suggestion
        push(initialComponent)
```

Reply:
Either `replace(null, ...)` or `clear+push`; both of them not needed

Reply:
Do we want to clear the stack here or not? The original code didn't do that

Reply:
yeah clear was not needed. I changed the init() to restart() as it makes more sense imo. Calling init twice would end up with two welcome pages. Restart just clears the content, what now is with no risk of clearing some shared stuff from the stack.